languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -54722,7 +54722,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                         <token>cameras</token>
                     </marker>
                 </pattern>
-                <message>Cameras est l'indicatif de "se camer". Vouliez-vous dire <suggestion>caméras</suggestion> ?</message>
+                <message>Cameras est l'indicatif du verbe « se camer ». Vouliez-vous dire « <suggestion>caméras</suggestion> » ?</message>
```

Comment:
**Formatting:** Sorry, I meant on the Cameras in 
`<message>Cameras est l'indicatif du verbe « se camer ».`